### PR TITLE
Add tags to UserTrigger module

### PR DESCRIPTION
**Pull request by IvanIvanoff**
#### Summary

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

File: lib/sanbase/signals/user_trigger.ex [hunk 1]
```diff
@@ -88,9 +126,13 @@ defmodule Sanbase.Signals.UserTrigger do
       preload: [{:user, :user_settings}]
```
Comment (tspenov) [inline lib/sanbase/signals/user_trigger.ex]: we should preload tags here

File: lib/sanbase/signals/user_trigger.ex [hunk 2]
```diff
@@ -88,9 +126,13 @@ defmodule Sanbase.Signals.UserTrigger do
...
+    |> Enum.map(&trigger_in_struct/1)
   end
 
+  @doc ~s"""
+  Create a new user trigger that is used to fire signals.
+  To create a new trigger `settings` and `title` parameters must be present
+  """
   @spec create_user_trigger(%User{}, map()) ::
           {:ok, %__MODULE__{}} | {:error, String.t()} | {:error, %Ecto.Changeset{}}
   def create_user_trigger(%User{id: user_id} = _user, %{settings: settings} = params) do
```
Comment (tspenov) [inline lib/sanbase/signals/user_trigger.ex]: we should preload tags on `create_user_trigger`

Comment (IvanIvanoff) [reply lib/sanbase/signals/user_trigger.ex]: https://github.com/santiment/sanbase2/pull/987/files#diff-0242dcfda9cd696d35ce332c83bd7b54R326
This test works

File: lib/sanbase/signals/user_trigger.ex (hunk 1 above)
Comment (IvanIvanoff) [reply lib/sanbase/signals/user_trigger.ex]: This is not exposed in the public API, only to the evaluator, which does not need the tags (for now)

File: test/support/Untitled-3 [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Първата лекция на курса „Функционално програмиране с Elixir“ ще се проведе утре, 19.02.2019 г., от 19:00  ч. в зала 02.
```
Comment (tspenov) [inline test/support/Untitled-3]: :D

Comment (tspenov) [inline test/support/Untitled-3]: 😄

Comment (IvanIvanoff) [reply test/support/Untitled-3]: ooooops

**Pull request merged**